examples/step-56/doc/intro.dox

```diff
@@ -0,0 +1,159 @@
+<i>This program was contributed by Ryan Grove and Timo Heister.</i>
+
+<a name="Intro"></a> 
+<h1>Introduction</h1>
+
+<h3> Stokes Problem </h3>
+
+The purpose of this tutorial is to create an efficient linear solver
+for the Stokes equation and compare it to alternative
+approaches. Using FGMRES with geometric multigrid as a preconditioner
+for the velocity block, we see that the linear solvers used in step-22
+cannot keep up since multigrid is the only way to get $O(n)$ solve
+time. Using the Timer class, we collect some statistics to compare
```

Comment:
This sounds slightly funny -- as if FMGRES+GMG is actually used in step-22. How about: "Here, we will use FGMRES ... velocity block, and we will show in the results section that this is a fundamentally better approach than the linear solvers used in step-22 since multigrid..."